extensions/console/02_console.js

```diff
@@ -195,8 +195,8 @@
       const needed = (columnWidths[i] - len) / 2;
       // round(needed) + ceil(needed) will always add up to the amount
       // of spaces we need while also left justifying the output.
-      out += `${StringPrototypeRepeat(" ", needed)}${cell}${
-        StringPrototypeRepeat(" ", MathCeil(needed))
+      out += `${cell}${StringPrototypeRepeat(" ", MathCeil(needed))}${
```

Comment:
This can be done much cleaner now that we are left-aligning.

line 195:
```js
const needed = columnWidths[i] - len;
```
line 198:
```js
out += `${cell}${StringPrototypeRepeat(" ", needed)}`;
```